src/snoowrap.js

```diff
@@ -78,8 +78,9 @@ const snoowrap = class snoowrap {
     if (!userAgent && !isBrowser) {
       return requiredArg('userAgent');
     }
-    if (!accessToken &&
-        (clientId === undefined || clientSecret === undefined || refreshToken === undefined) &&
+    if ((!accessToken || typeof accessToken !== 'string') &&
+        (clientId === undefined || clientSecret === undefined || refreshToken === undefined
+            || typeof refreshToken !== 'string') &&
```

Comment:
It seems like `refreshToken === undefined` is now redundant with the new check.

Reply:
True, removed the `refreshToken === undefined` check